Move perfy stuff into Perf Diagnostics report aggregation.

paulirish opened the pull request:
screenshot:
![image](https://cloud.githubusercontent.com/assets/39191/22956486/2b8a1190-f2d7-11e6-9567-81d6dffcb576.png)

ebidel commented on lighthouse-core/audits/dobetterweb/script-blocking-first-paint.js:
```diff
@@ -33,7 +33,7 @@ class ScriptBlockingFirstPaint extends Audit {
       category: 'Performance',
       name: 'script-blocking-first-paint',
-      description: 'Avoids `<script>` in head that delay first paint',
+      description: 'Render-blocking `<script>`',
```
should we be consistent? `<script>` -> scripts?

ebidel commented on lighthouse-core/audits/byte-efficiency/uses-optimized-images.js:
```diff
@@ -40,7 +40,7 @@ class UsesOptimizedImages extends Audit {
       category: 'Images',
       name: 'uses-optimized-images',
-      description: 'Avoids unoptimized images',
+      description: 'Unoptimized Images',
```
Images -> image

sentence case everywhere.  PSI does that too.

<img width="528" alt="screen shot 2017-02-14 at 7 01 26 pm" src="https://cloud.githubusercontent.com/assets/238208/22959068/0b962138-f2e8-11e6-9f1d-69574a8c4546.png">

ebidel commented on lighthouse-core/config/default.json:
```diff
@@ -372,20 +359,19 @@
     }]
   }, {
-    "name": "Performance Metrics",
+    "name": "Performance diagnostics",
```
would "Performance" be enough?

patrickhulce commented on lighthouse-core/audits/byte-efficiency/unused-css-rules.js:
```diff
@@ -29,7 +29,7 @@ class UnusedCSSRules extends Audit {
       category: 'CSS',
       name: 'unused-css-rules',
-      description: 'Avoids loading unnecessary CSS',
+      description: 'Unused CSS Rules',
```
Haha, yeah might be worth having a dedicated report messaging discussion just to iron out what we want to standardize on :)

patrickhulce commented on lighthouse-core/config/default.json:
```diff
@@ -372,20 +359,19 @@
-          "expectedValue": 0,
-          "weight": 1
-        }
+        "unused-css-rules": {},
```
perf.json already looks like this, correct?

paulirish commented:
PTAL. Updated.

Also updated perf.json as `--perf` is currently broken.

@ebidel If you're down, I'd like to file a ticket to address our green-checks-next-to-"unused css rules" bug in the report. I've already added it to [Lighthouse Report Requirements](https://docs.google.com/document/d/1A2JvCi6cQwHKB7MI_vTqTUF97xsimKQOb-KaOXFsWAQ/edit)

ebidel commented:
IMO we should keep "Avoids..." until we've figured out something different. That language was hashed out over a  couple of PRs and had Kayce input. 

I'm on mobile, but there's a Pavel ticket already open that you can add to.

paulirish commented:
Merged to fix #1775 and others. 

Will comment on #1613 and take the bug to address wording. Also #1750 by @sendilkumarn will help considerably with the icon/text combo.
